cli/src/main/java/com/box/l10n/mojito/cli/command/RepoCreateCommand.java

```diff
@@ -35,6 +35,9 @@
     @Parameter(names = {Param.REPOSITORY_DESCRIPTION_LONG, Param.REPOSITORY_DESCRIPTION_SHORT}, arity = 1, required = false, description = Param.REPOSITORY_DESCRIPTION_DESCRIPTION)
     String descriptionParam;
 
+    @Parameter(names = {Param.CHECK_SLA_LONG, Param.CHECK_SLA_SHORT}, arity = 1, required = false, description = Param.CHECK_SLA_DESCRIPTION)
+    String checkSLA;
```

Comment:
You can use a `Boolean` instead of a `String` and put a default value to it. `Boolean checkSLA = false` this will also document the default value.

Reply:
use Boolean checkSLA = false instead.